<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ticket</class>
 <widget class="QDockWidget" name="Ticket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>450</width>
      <height>731</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="manage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manage Matro</string>
      </property>
     </widget>
     <widget class="QLabel" name="manimg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="sub">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Subsctiption</string>
      </property>
     </widget>
     <widget class="QLabel" name="subimg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>49</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="station">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>560</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Station</string>
      </property>
     </widget>
     <widget class="QLabel" name="staimg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>49</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="metroimg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>670</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>690</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BAXAUTRIAL</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Egypt Metro</string>
     </property>
    </widget>
    <widget class="QLabel" name="adminimg">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="manage_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Home Page</string>
      </property>
     </widget>
     <widget class="QLabel" name="imghome">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>49</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #BDBDBD;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #BDBDBD;
border-radius: 12;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tickets</string>
      </property>
     </widget>
     <widget class="QLabel" name="ticimg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>49</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>680</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BAXAUTRIAL</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFC200;
border-radius: 12;
color: black;</string>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ticket Cost</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>216</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tire #1 Cost :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tire #2 Cost :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tire #3 Cost :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="mail">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="mail_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>320</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="mail_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>560</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFC200;
border-radius: 12;
color: black;</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
